### PR TITLE
[prometheus-vmware-rules] Revise Cinder alerts

### PR DESCRIPTION
**Pull request by kevin-fischer**
- Revamps the metric names from the DatastorePropertiesCollector in the ConfigMap to adhere to a more generic form and align with the current formatting conventions used in the codebase/repository
- Updates the expressions, metadata, description, and summary

### CODE REVIEW

**Comment by kevin-fischer** [comment]
@max-len @richardtief

**Review by himanip94 — APPROVED** [review]
lgtm

File: system/vmware-monitoring/templates/collector-configmap.yaml [hunk 1]
```diff
@@ -146,9 +146,9 @@ data:
...
       - metric_suffix: "hostcount"
         key: "datastore|hostcount"
-      - metric_suffix: "cinder_tag"
+      - metric_suffix: "summary_tag"
         key: "summary|tagJson"
-      - metric_suffix: "cinder_state"
+      - metric_suffix: "summary_custom_tag_cinder_state"
         key: "summary|customTag:cinder_state|customTagValue"
 
     SDRSStatsCollector:
```
Comment (richardtief) [inline system/vmware-monitoring/templates/collector-configmap.yaml:154]: Please deploy this change first to verify that the metrics are available as you expect.

**Review by richardtief — COMMENTED** [review]
I have not checked the content. Just a small hint of a change.

**Review by max-len — APPROVED** [review]

File: system/vmware-monitoring/templates/collector-configmap.yaml (hunk 1 above)
Comment (kevin-fischer) [reply system/vmware-monitoring/templates/collector-configmap.yaml:154]: Thx Richard.

Sure I'll roll out the vmware-monitoring pipeline first.

**Pull request merged**